opentelemetry-api/test-requirements.txt

```diff
@@ -1,6 +1,6 @@
 asgiref==3.7.2
 Deprecated==1.2.14
-importlib-metadata==6.11.0
+importlib-metadata<9.0.0
```

Comment:
We tend to keep this fixed, does it work locally?

Reply:
Yes, tested it with Python 3.8 through 3.12 and all green.

Reply:
Please keep it the way it is or just bump to a fixed version for reproducibility

Reply:
ok